restful_api/myapi/api/schemas/__init__.py

```diff
@@ -1,20 +1,19 @@
 from myapi.api.schemas.user import UserSchema
 from myapi.api.schemas.employee import EmployeeSchema
-from myapi.api.schemas.official_enroll import OfficialEnrollSchema
+from myapi.api.schemas.enroll import EnrollSchema
```

Comment:
Did u import twice? Hm should I start adding a linter?

Reply:
Fixed - yes please haha. I think a linter would be good.